### PR TITLE
Ps 29 sending cookie with request

### PR DESCRIPTION
**Pull request by JafariM**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -18,7 +19,10 @@ function App() {
   const [message, setMessage] = useState('');
   const { status } = useAuth();
   const isLoggedIn = status === AuthStatus.LoggedIn;
-  // const navigate = useNavigate();
+  // create an axios instance
+  const instance = axios.create({
+    withCredentials: true,
+  });
```
Comment (akosasante) [inline src/App.js]: Is this being used, in this file?

File: src/App.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import Home from './components/Home';
 import { Routes, Route, Navigate } from 'react-router-dom';
 import Logout from './components/Logout';
 import Navbar from './components/Navbar';
+import axios from 'axios';
```
Comment (akosasante) [inline src/App.js]: Probably can remove along with lines 22-25

File: src/components/Home.js [hunk 3]
```diff
@@ -1,8 +1,27 @@
...
+        setData(response.data); // Set the state with the response data
+        console.log('HTTP response data:', response.data);
+      })
+      .catch((error) => {
+        console.error('Error fetching data:', error);
+      });
+  }, []);
+
+  // end of testing
+
```
Comment (akosasante) [inline src/components/Home.js]: Tested, it is working; feel free to remove this bit before merging :+1:

**Review by akosasante — APPROVED** [review]
A couple things to remove, then LGTM :rocket: :ship:

File: src/App.js (hunk 1 above)
Comment (JafariM) [reply src/App.js:25]: I don't have this lines in my App.js anymore, I have no idea how still it appears in this branch

Comment (JafariM) [reply src/App.js]: No it does not. I will remove it

File: src/components/Home.js (hunk 3 above)
Comment (JafariM) [reply src/components/Home.js]: I removed that

File: src/App.js (hunk 2 above)
Comment (JafariM) [reply src/App.js]: Removed

**Pull request merged**